### DISCUSSION TITLE
@include() hot reload

### DISCUSSION THREAD

**Discussion by tadmi** [vuepress-theme-hope/vuepress-theme-hope#2686] [Q&A] [answered]
Please  add the possibility of hot reloading files that are added to the page by the **@include()**.

You have to overload the Vite to see the changes.

**Comment by Mister-Hope** [comment]
This can not be done as long as this pr is not being merged: https://github.com/vuepress/vuepress-next/pull/1228

I do not have authorize to merge pr to main or publish new versions, so as long as @meteorlxy leaves them, you shall wait.

Also, you can have a workaround by:

1. deleting the `@`  and save to hotreload
2. adding the `@` back then save to let the file being read again.

**Comment by Mister-Hope** [comment, marked as answer]
Track https://github.com/vuepress-theme-hope/vuepress-theme-hope/issues/2766

**Comment by tadmi** [comment]
@Mister-Hope 

On v2.0.0-beta.200 the includes are still not hot reloaded.

If included md file does not [end with a newline character](https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md047.md) then @include-pop() gets into the html after insertion:
![image](https://user-images.githubusercontent.com/63556979/228482738-2f70e491-ec01-4b5c-8375-9c62e67404e3.png)

**Reply by Mister-Hope** [reply]
please open 2 new issues about them.

I may forget this if you report problem in discussion

**Reply by tadmi** [reply]
stricter requirements for opening an issue

**Reply by Mister-Hope** [reply]
Give me a pr like https://github.com/mdit-plugins/mdit-plugins/commit/f98f0e0330752aa36d7b62e7c890ddd49d74a3c3 with failed tests to let me pick up this again.

**Reply by Mister-Hope** [reply]
![image](https://user-images.githubusercontent.com/33315834/229040313-9c609a2a-f3c9-4bda-9c25-2037f8883f74.png)

I am sure hotrealod working on my machine, this is a warning. Double check what you are reporting, open an issue about it with a reproduction and ensure you can reproduce it with reproduction steps you provided with any demo or template. I will ignore any of your reports without a reproduction starting from now, and you may get yourself block if you are spamming in this way. You are warned. This is opensource and I hate others wasting my time.

**Comment by tadmi** [comment]
~~Nested includes are not included ( in the included file)~~

**Reply by Mister-Hope** [reply]
Are you sure you set deep option to true?

**Reply by tadmi** [reply]
sorry, I was inattentive

**Reply by Mister-Hope** [reply]
This option is default false as the error stack is confusing if user include files circular
